frontend/src/metabase/parameters/utils/mapping-options.js

```diff
@@ -64,9 +64,15 @@ export function getParameterMappingOptions(
     return tagNames ? tagNames.map(buildTextTagOption) : [];
   }
 
-  if (isActionCard(card)) {
-    // Action parameters are mapped via click behavior UI for now
-    return [];
+  if (isActionDashCard(dashcard)) {
+    const actionParams = dashcard?.action?.parameters?.map(p => ({
+      icon: "variable",
+      isForeign: false,
+      name: p.id,
+      ...p,
+    }));
```

Comment:
I'd vote for avoiding such short variable names like `p`. `param` or `parameter` are more readable IMO